src/powershell-win-core/powershell-win-core.csproj

```diff
@@ -4,6 +4,7 @@
     <Description>PowerShell Core on Windows top-level project</Description>
     <AssemblyName>pwsh</AssemblyName>
     <OutputType>Exe</OutputType>
+    <TieredCompilation>true</TieredCompilation>
```

Comment:
I'd add this to our tool/test projects too.

Reply:
I can do that.